spec/models/post_mover_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require "rails_helper"
```

Comment:
This line line is [not necessary](https://github.com/discourse/discourse/blob/a2000a3559eef9ef17d83ee79cf9a3319eb5df24/.rspec#L1C4-L1C4).